<launch>
	<param name = "node_type"
		type = "str"
		value = "nb3_i7_udp_receiver" />
	<param name = "write_rate"
		type = "int"
		value = "100" />
	<param name = "ADDR_LOCAL"
		type = "str"
		value = "10.10.10.1" />
	<param name = "ADDR_OTHER"
		type = "str"
		value = "10.10.10.4" />
	<param name = "PORT_LOCAL"
		type = "int"
		value = "9007" />
	<param name = "PORT_OTHER"
		type = "int"
		value = "9008" />
	<node name = "nb3_i7_udp_receiver"
		pkg = "hw_interface"
		type = "Comm_Node" />
</launch>
